### ISSUE TITLE
Cannot read property 'success' of undefined

### ISSUE THREAD

**Issue by scx567888** [element-plus/element-plus#722]
```
<template>
  <el-dropdown appendToBody trigger="hover" @command="handleSetLanguage">
    <div>
      <scx-icon icon="language"/>
    </div>
    <template v-slot:dropdown>
      <el-dropdown-menu>
        <el-dropdown-item :disabled="language==='zh-cn'" command="zh-cn">
          简体中文
        </el-dropdown-item>
        <el-dropdown-item :disabled="language==='en'" command="en">
          English
        </el-dropdown-item>
      </el-dropdown-menu>
    </template>
  </el-dropdown>
</template>

<script>
import {changeSetting, state} from "../../store";
import {t} from "../../i18n";
import {computed} from "vue";
import {ElMessage} from 'element-plus'

export default {
  name: 'scx-lang-select',
  setup() {
    function handleSetLanguage(nowLang) {
      changeSetting('language', nowLang);
      ElMessage.success(t('settings.setLanguageSuccess'));
    }

    const language = computed(() => state.settings.language);
    return {handleSetLanguage, language}
  }
}
</script>
```
package.json
```
  "dependencies": {
    "element-plus": "^1.0.1-beta.0",
    "vue": "^3.0.3",
    "vue-router": "^4.0.0-rc.5"
  },
  "devDependencies": {
    "vite": "^1.0.0-rc.13",
  }

```
main.js
```
import {createApp} from 'vue';
import App from './App.vue'; 
import ElementPlus from 'element-plus'; //element
import 'element-plus/lib/theme-chalk/index.css'; //element 的 样式表

createApp(App)
    .use(ElementPlus)
    .mount('#app');

```
is element-plus bug or vite bug?

**Comment by Ryan2128** [comment]
Are you using the latest version? It works well in the latest version.

[DEMO](https://codesandbox.io/s/priceless-benz-yey3y?file=/src/App.vue)

**Comment by scx567888** [comment]
> Are you using the latest version? It works well in the latest version.
> 
> [DEMO](https://codesandbox.io/s/priceless-benz-yey3y?file=/src/App.vue)

thank you ; i try use webpack ; is ok;  but when i use vite ; is undefined; so I'm not certain is a element-plus bug or vite bug;

**Comment by JeremyWuuuuu** [comment]
> > Are you using the latest version? It works well in the latest version.
> > [DEMO](https://codesandbox.io/s/priceless-benz-yey3y?file=/src/App.vue)
> 
> thank you ; i try use webpack ; is ok; but when i use vite ; is undefined; so I'm not certain is a element-plus bug or vite bug;
@scx567888 
Hi, thanks for reporting this issue. If you wanted to use it with `Vite` then explicit importing is required since the `index.js` from `element-plus/root` is built in UMD format. You can try this:

```ts
import ElMessage from 'element-plus/lib/el-message'
```

**Comment by JeremyWuuuuu** [comment]
This should be fixed in #577 , now `Vite` knows where to find the components. This will be released in `Beta1`, so I am closing this, it won't take long to publishing the new version, you can then run `yarn upgrade` to try it out.
